tests/php/PHPStanBearTest.py

```diff
@@ -0,0 +1,23 @@
+from bears.php.PHPStanBear import PHPStanBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+good_file = """
+<?php
+    $a = 1;
+?>
+"""
+
+
+bad_file = """
+<?php
+    $a =
+?>
+"""
+
+
+PHPStanBearTest = verify_local_bear(
```

Comment:
So using `check_results` won't harm because it is more rigorous then `check_validity`